pipeline/relation_embedder/relation_embedder/src/main/scala/weco/pipeline/relation_embedder/lib/StdInStrings.scala

```diff
@@ -0,0 +1,34 @@
+package weco.pipeline.relation_embedder.lib
+
+import org.apache.pekko.NotUsed
+import org.apache.pekko.stream.IOResult
+import org.apache.pekko.stream.scaladsl.{
+  Flow,
+  Framing,
+  Source,
+  StreamConverters
+}
+import org.apache.pekko.util.ByteString
+
+import scala.concurrent.Future
+
+trait StdInStrings {
+  private val stdinSource: Source[ByteString, Future[IOResult]] =
```

Comment:
I understand this is to accommodate the interfaces we've woven into the app elsewhere, but it does neatly highlight how what could be "take a string, parse it, hand that to the app" is somewhat obfuscated by using Akka Flow.

Reply:
Yes.  Part of the reason I pulled it out like this is that I think it makes it easier to think about those kinds of changes.

However, now that I have, and now you mention it, I think it would be pretty simple for me to replace this with a io.Source and only turn it into Akka Source just before it enters the bit that's already using Flow.  I'm going to give that a go right now.